test/jdk/java/io/BufferedInputStream/LargeCopyWithMark.java

```diff
@@ -23,65 +23,43 @@
 
 /* @test
  * @bug 7129312
+ * @requires (sun.arch.data.model == "64"  & os.maxMemory > 4g)
  * @summary BufferedInputStream calculates negative array size with large
  *          streams and mark
  * @library /test/lib
```

Comment:
Looks like this can be removed. Otherwise looks good.

Reply:
Yes, you are correct: fixed. Thanks.